hoomd/md/pytest/test_meshpotential.py

```diff
@@ -157,15 +185,55 @@ def test_after_attaching(triplet_snapshot_factory, simulation_factory,
         mesh_bond_potential.mesh = mesh1
 
 
+@pytest.mark.parametrize("mesh_bond_cls, potential_kwargs",
+                         get_mesh_bond_and_args())
+def test_multiple_types(triplet_snapshot_factory, simulation_factory,
+                        mesh_bond_cls, potential_kwargs):
+
+    sim = simulation_factory(triplet_snapshot_factory(d=0.969, L=5))
```

Comment:
```suggestion
    sim = simulation_factory(mesh_snapshot_factory(d=0.969, L=5))
```
?

Reply:
I think `triplet_snapshot_factory ` is correct